server/test/auth/signup.spec.js

```diff
@@ -0,0 +1,18 @@
+import chai, { expect } from 'chai';
```

Comment:
I'd like to see more edge cases tested;

- negative testing
- make more comprehensive checks, aside from status code and token.

Reply:
@waleCloud I am not supposed to work on validating the input. Could you please shed more light on *negative testing* in this regard?

I would add further checks on user details like first name, last name and email

Reply:
Negative testing is a scenario where you supply wrong values/data inputs to the application or service you've built to see how well it handles it.

A wrong/invalid email, for example, should throw some kind of error.

Reply:
Why are you not suppose to work on validating the input?

Reply:
Thanks for the clarification. There's a separate PT story for validating signup input and it's assigned to @davidokonji. You could confirm this here [#164798906](https://www.pivotaltracker.com/story/show/164798906)